test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1873,9 +1877,10 @@ def __init__(self) -> None:
             def forward(self, x):
                 return self.bn(x)
 
-        if TEST_CUDA:
-            m = M().train().cuda()
-            example_inputs = (torch.randn(1, 3, 3, 3).cuda(),)
+        if TEST_CUDA or TEST_HPU:
```

Comment:
is this the only test you need to add hpu?

Reply:
currently we are adapting only those tests which are run on CUDA devices.  Not adapting the CPU only tests.